test.cmake

```diff
@@ -108,6 +108,9 @@ MACRO(ADD_PYTHON_UNIT_TEST NAME SOURCE)
   SET(MODULES "${ARGN}")  # ARGN is not a variable
   FOREACH(MODULE_PATH IN LISTS MODULES)
     LIST(APPEND PYTHONPATH "${CMAKE_BINARY_DIR}/${MODULE_PATH}")
+    IF(WIN32)
+      LIST(APPEND PYTHONPATH "${CMAKE_BINARY_DIR}/${MODULE_PATH}/$<CONFIG>")
+    ENDIF(WIN32)
```

Comment:
shouldn't there be a `else` here ?

Reply:
Not in the way we are generating the Python files currently.